src/containers/Dashboard.js

```diff
@@ -1623,7 +1623,17 @@ class Dashboard extends PureComponent {
     const { name, value } = e.target;
     var errors = this.state.errors;
     errors[`${name}Error`] = "";
-    this.setState({ [name]: value, errors: errors });
+    if (name === "estimate") {
+      if (value.split(".")[0] === "" &&  Number(value) > 0) {
```

Comment:
Why we have so much nesting to implment this  as eventually you are setting the value only in two cond. 

others it is 
this.setState({ [name]: value, errors: errors });

Reply:
1. If user enters '0.4567' and delet the no's and the decimal then the estimation time with 0 can be added, which is not allowed thats why added the second condition.
2.  if user enter the estimation time value then it shouldn't affect other values.